runtime/src/main.rs

```diff
@@ -38,6 +41,20 @@ const BANNER: &str = r#"
 #[allow(clippy::empty_loop)]
 pub extern "C" fn entry_point() -> ! {
     cprintln!("{}", BANNER);
+
+    #[cfg(target_arch = "riscv32")]
+    unsafe {
+        // Write meivt (External Interrupt Vector Table Register)
+        // VeeR has been instantiated with RV_FAST_INTERRUPT_REDIRECT,
+        // so external interrupts always bypass the standard risc-v dispatch logic
+        // and instead load the destination address from this table in DCCM.
+        core::arch::asm!(
+            "la {tmp}, _ext_intr_vector",
+            "csrw 0xbc8, {tmp}",
+            tmp = out(reg) _,
+        );
```

Comment:
nit: why do this here instead of the startup script?

Reply:
startup script is shared between fmc and runtime. We want this logic in the runtime only.

Reply:
Also, since #1141, the FMC overflows 16k with the added 160 bytes of the lookup table....